.github/workflows/build-hw-reusable.yml

```diff
@@ -87,13 +87,18 @@ jobs:
         -DUR_STATIC_ADAPTER_${{matrix.adapter.name}}=${{matrix.adapter.static_adapter}}
         -DUR_DPCXX=${{github.workspace}}/dpcpp_compiler/bin/clang++
         -DUR_SYCL_LIBRARY_DIR=${{github.workspace}}/dpcpp_compiler/lib
+        -DCMAKE_INSTALL_PREFIX=${{github.workspace}}/install
         ${{ matrix.adapter.name == 'HIP' && '-DUR_CONFORMANCE_AMD_ARCH=gfx1030' || '' }}
         ${{ matrix.adapter.name == 'HIP' && '-DUR_HIP_PLATFORM=AMD' || '' }}
 
     - name: Build
       # This is so that device binaries can find the sycl runtime library
       run: cmake --build ${{github.workspace}}/build -j $(nproc)
 
+    - name: Install
+      # This is to check that install command does not fail
+      run: cmake --install ${{github.workspace}}/build
+
```

Comment:
Ideally we'd check output against a list of expected files.

Reply:
I object. Too much logic to put in the test. Not worth doing it. Libs can be static or dynamic. Adapters may be built or not depending on cmake params. There are 100+ files installed, so check will have to selective and thus not complete.

If any client will depend on make install we can reconsider. Now the only client is benchmark framework.

Reply:
ok, makes sense.